include/can/core/messages.hpp

```diff
@@ -204,7 +204,7 @@ struct ReadFromEEPromResponse : BaseMessage<MessageId::read_eeprom_response> {
             std::min(data_length,
                      static_cast<eeprom::types::data_length>(limit - iter)),
             iter);
-        return iter - body;
+        return limit - body;
```

Comment:
Shouldn't this still be `iter`?

Reply:
The size of the message must always be the same. Regardless of the number of bytes read, we send the complete buffer.